src/components/ProductCard/ProductCard.jsx

```diff
@@ -25,16 +30,16 @@ const ProductCard =()=>{
           </div>
           
          <div className='card-container'>
-             {data &&
-             data.map((product) => (
+         {sortedProducts.map(({img,author,title,rating,price}) => (
                  
           <div className="vertical-ecomm">
-          <img className="img-vertical" src={product.img} alt="iphone 13 pro max" />
+          <img className="img-vertical" src={img} alt="iphone 13 pro max" />
           <p className="brand-name">
-            {product.author}<i className="far fa-heart rightheart"></i>
+            {author}<i className="far fa-heart rightheart"></i>
           </p>
-          <p className="brand-info">{product.title}</p>
-          <p className="price brand-info">{product.price}</p>
+          <p className="brand-info">{title}</p>
+          <p className='font-s'>Rating: {rating} ⭐</p>
+          <p className="price brand-info">{price}</p>
```

Comment:
Add rupee symbol

Reply:
Thanks for pointing it out, will work on that.